dotcom-rendering/src/components/SeriesSectionLink.tsx

```diff
@@ -179,6 +180,8 @@ export const SeriesSectionLink = ({
 	badge,
 	isMatch,
 }: Props) => {
+	const { renderingTarget } = useConfig();
+	const showBadge = renderingTarget === 'Web' && badge !== undefined;
```

Comment:
```suggestion
	const showBadge = renderingTarget === 'Web' && !isUndefined(badge);
```

Reply:
What would you say is the advantage of a type predicate over conventional TS narrowing here?

Reply:
I would have said “consistency”, but realised that `isUndefined` is only used 30 times in 13 files whereas strict equality with `undefined` is used 140 times in 91 files.

I do think we should pick one approach and use it consistently… but this need not be addressed in this PR.